docs/aiops-projects.md

```diff
@@ -0,0 +1,41 @@
+# AI Ops Projects 
+
+_This document contains a list of active projects within the AI Ops Team_
+
+1. **OCP Alert Prediction** : If a customer’s OpenShift cluster goes down, it can have a significant impact on their business. Since there are a variety of reasons why an OpenShift cluster might fail, finding and fixing the issue that the cluster suffers from is not always trivial. However, if we can predict in advance whether a cluster will run into a given issue, then we may be able to fix it before it fails or before it severely impacts the customer. Issues in a cluster are often defined by, or closely related to, the alerts that it fires. So predicting alerts can be a step towards predicting the underlying issue. Thus, the goal of this project is to predict whether a cluster will fire a given alert within the next hour. 
+
+2. **Categorical Encoding** : Experiments to show how to use encoding schemes for nominal categorical variables in Red Hat data with unsupervised learning problems.
+
+3. **OCP4 Anomaly Detection** : OCP4 deployments can suffer from a number of different issues and bugs. It can be tedious for an engineer to inspect and diagnose each deployment individually, which in turn can adversely affect customer experience. In this project, we work on the following two initiatives to address this problem.
+    * Anomaly Detection: In this approach, we try to identify issues before they occur, or before they significantly impact customers. To do so, we find deployments that behave “anomalously” and try to explain this behaviour.
+    * Diagnosis Discovery: In this approach, we try to identify deployments that exhibit similar “symptoms” (issues), and determine exactly what makes these deployments similar to one another. The support engineer can then use this information to determine the “diagnosis” of the issues, and apply the same or similar fix to all the deployments.
+
+4. **Data Science Workflows** : AI Ops team has been working on developing a more structured process around how we manage, execute and deliver on our data science projects, especially those where we collaborate with other Red Hat teams. Having a common framework that we can all start to build from as the team grows and continues to take on more data science projects, it will be hugely beneficial to have an agreed upon and documented process like this in place. And more important than just the existence of some documentation, is that we actually use these tools and find that they provide us with some value. Meaning, that we should keep updating and evolving this process to suit our needs. To do that we are very interested in hearing your thoughts and getting your feedback (and ideally adoption) on this data science project workflow to see where it can be improved, streamlined or simply be made clearer. 
```

Comment:
please remove the last sentence as it is award here: "To do that we are very interested in hearing your thoughts and getting your feedback (and ideally adoption) on this data science project workflow to see where it can be improved, streamlined or simply be made clearer. "